x/interchainquery/keeper/msg_server.go

```diff
@@ -54,7 +55,7 @@ func (k msgServer) SubmitQueryResponse(goCtx context.Context, msg *types.MsgSubm
 	noDelete := false
 	// execute registered callbacks.
 
-	keys := []string{}
+	keys := make([]string, len(k.callbacks))
```

Comment:
`make` a slice of string x long, and then appending x strings, will give you a slice of 2x length, with x empty elements, iirc.

Reply:
https://go.dev/play/p/aUOx9vQmUr_m?v=goprev

Reply:
Will likely panic on line 67 for referencing a non-existent element